flixel/text/FlxText.hx

```diff
@@ -775,7 +787,13 @@ class FlxText extends FlxSprite
 			newHeight = oldHeight;
 		}
 		
+		_flashRect.setTo(0, 0, newWidth, newHeight);
+		
+		#if flash11
```

Comment:
Why is this `#if flash11`? Flash Player version 11+ doesn't necessarily mean that you're using Stage3D rendering. I think this should be `#if (flash11 && FLX_RENDER_TILE)`. 
Since you're gonna need that in a number of places, I recommend creating a helper define in `FLX_DEFINES` for this (`FLX_RENDER_STAGE3D` maybe ?).